crates/algokit_transact_ffi/src/transactions/application_call.rs

```diff
@@ -54,7 +54,7 @@ pub struct ApplicationCallTransactionFields {
 
     /// List of accounts in addition to the sender that may be accessed
     /// from the application's approval program and clear state program.
-    account_references: Option<Vec<Address>>,
+    account_references: Option<Vec<Account>>,
```

Comment:
Should these be string now, as it's valid to use a multisig address?

Reply:
Yes definitely, logisigs too